packages/ui5-middleware-livereload/lib/livereload.js

```diff
@@ -67,13 +67,31 @@ module.exports = async ({ resources, options }) => {
     if (options.configuration && options.configuration.debug) {
         debug = options.configuration.debug;
     }
+        
+    let serverOptions = {
+        debug: debug,
+        extraExts: extraExts ? extraExts.split(",") : undefined,
+        port: port,
+        exclusions: exclusions
+    };
+    
+    const cli = require("yargs");
+    if (cli.argv.h2) {
+        const path = require("path");
```

Comment:
`path` is already required https://github.com/ui5-community/ui5-ecosystem-showcase/blob/8f1ff9d76a5f8ef85db4af62748cc7f5ec456011/packages/ui5-middleware-livereload/lib/livereload.js#L3, so no need to do it again

Reply:
Good catch, I didn't observe it.